lib/context/auth/auth.context.tsx

```
@@ -0,0 +1,77 @@
+import { ApolloError } from "@apollo/client";
+
+// Core
+import useToast from "@/lib/hooks/useToast";
+
+// Context
+import { useConfig } from "../configuration/configuration.context";
+
+// Google API
+import { GoogleOAuthProvider } from "@react-oauth/google";
+
+const AuthContext = createContext({} as {});
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)